src/components/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,25 @@
+import { ChangeEventHandler, Dispatch, FormEventHandler, SetStateAction, useState } from 'react';
+import thr from '../../helpers/throttle';
+import { StyledForm, StyledIconButton, StyledInput } from './styles';
+
+const throttle = thr();
```

Comment:
please, install 'lodash/throttle' and import it from the library.

Reply:
Will do